### ISSUE TITLE
Fix architecture of `vcpkg` packages

### ISSUE THREAD

**Issue by ethanc8** [ethanc8/titanian-repo#1]
Unfortunately, the architecture of the `vcpkg` packages is `all`, which is not actually correct.

**Comment by ethanc8** [comment]
The `amd64` package has been fixed in [commit 94dabf0](https://github.com/ethanc8/titanian-repo/commit/94dabf082a5ac7071a44ff11cb5f952e05cc6c12)

**Comment by ethanc8** [comment]
I applied various other fixes to the packages.
```bash
chmod -R a+rwX debian/opt/vcpkg
mkdir -p debian/usr/bin
ln -s "/opt/vcpkg/vcpkg" debian/usr/bin/vcpkg
mkdir -p debian/etc/profile.d # On non-amd64 systems
echo "export VCPKG_FORCE_SYSTEM_BINARIES=true" > debian/etc/profile.d/vcpkg.sh # On non-amd64 systems
```
